### PR TITLE
change_status_fix

### PR DESCRIPTION
**Pull request by Noor-mommadhi**
Fixes -#8180

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -531,6 +531,7 @@
     //===================================================================================
     Route::middleware('auth')->group(function () {
         Route::get('client-profile', [Client\helpdesk\GuestController::class, 'getProfile'])->name('client.profile'); /*  User profile get  */
+        Route::post('select_all', [Agent\helpdesk\TicketController::class, 'select_all'])->name('select_all');
```
Comment (RafficMohammed) [inline routes/web.php:534]: use proper route name like select/all

File: resources/views/themes/default1/client/helpdesk/ckeckticket.blade.php [hunk 2]
```diff
@@ -1,5 +1,6 @@
 @extends('themes.default1.client.layout.client')
```
Comment (RafficMohammed) [inline resources/views/themes/default1/client/helpdesk/ckeckticket.blade.php:1]: Write a test cases for this P.R

**Review by RafficMohammed — CHANGES_REQUESTED** [review]

**Comment by DhanyaChengappa** [comment]
- [ ] Add a confirmation pop-up before status change to close

**Comment by Noor-mommadhi** [comment]
@DhanyaChengappa please create a separate issue for "Add a confirmation pop-up before status change to close"

**Comment by Sabaric12** [comment]
@Noor-mommadhi get demo approved and code approved for this PR

**Comment by Sabaric12** [comment]
@Noor-mommadhi resolve the conflicts

**Review by RafficMohammed — APPROVED** [review]

**Comment by Sabaric12** [comment]
Round 1: Testing Summary
Tested this PR with the existing installation the mentioned issues are working fine now
Total time in days: 1.5 Hours
Total issues:0
Minor - 0 
Major - 0 
Critical - 0
No of times PR was sent for Functionality correction:0
PR breaks existing functionality: No
Note:-
Separate Issues are created for the pop-up messages
https://github.com/ladybirdweb/faveo-helpdesk/issues/8247

**Review by Sabaric12 — DISMISSED** [review]

**Comment by AbishekChinnaraju** [comment]
**Second Round  Testing Summary :**
Tested this PR with the existing server installation 
Total time in days: 1.5 Hours
Total issues:0
Minor - 0
Major - 0
Critical - 0
No of times PR was sent for Functionality correction:0
PR breaks existing functionality: No
**Note:-**
Separate Issues are created for the pop-up messages
https://github.com/ladybirdweb/faveo-helpdesk/issues/8247

**Review by AbishekChinnaraju — DISMISSED** [review]

**Comment by KNaveenraj-ladybird** [comment]
Hello ladybirdweb/faveo-helpdeskThank you for contacting us. This is an automated response confirming the receipt of your ticket. Our team will get back to you as soon as possible. When replying, please make sure that the ticket ID is kept in the subject so that we can track your replies.Ticket ID: AAAA-0000-0008  You can check the status of or update this ticket online at: https://localhost/faveo-helpdesk/public/check_ticket/eyJpdiI6IkxkV0pQNGZRaW9OZ09IMmtHTlBMVGc9PSIsInZhbHVlIjoiaEQzQXgzMjl1TGJrYzlJai9JWk9xUT09IiwibWFjIjoiYTcwODczY2E4YThkZmI5ZDZjYTAwMDgyMmEyMWQ2MWZiMjc4YjYwYWVjZDFhYzFjNTU3NWQ4MjNhZTI4MzlkNiIsInRhZyI6IiJ9

**Comment by KNaveenraj-ladybird** [comment]
---Reply above this line--- Hello ***@***.***,A reply been made to ticket AAAA-0000-0008FromName: NaveenrajE-mail: ***@***.*** Hello Naveenraj,New ticket AAAA-0000-0008created FromName: ladybirdweb/faveo-helpdesk   E-mail: ***@***.*** 
Closed #8224.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were assigned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/ladybirdweb/faveo-helpdesk/pull/8224#event-10776736179",
"url": "https://github.com/ladybirdweb/faveo-helpdesk/pull/8224#event-10776736179",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]Kind Regards, Support Center
Kind Regards, Support Center

**Pull request merged**